docs/First-Installation.md

```diff
@@ -113,49 +109,34 @@ For more information, see [Hotkeys](RetroArch-Configuration#hotkeys)
 
-## AUDIO
+## Audio
 
-In general RetroPie audio will work out of the box without any tweaking, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit the [Sound Issues Page](Sound-Issues) if you are using a USB Audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
+In general RetroPie audio will work out-of-the-box without any tweaking if using HDMI, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit this page if you are using a USB audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
 
 ## PLAY!
 
-After you've added your roms you need to restart EmulationStation in order for them to show up. You can restart EmulationStation from the start menu, or by rebooting your pi with `sudo reboot`. 
+After you've transferred your ROMs, you need to restart EmulationStation in order for them to show up. You can restart EmulationStation by pressing Start > Quit > Restart EmulationStation, or with SSH access by rebooting your pi with `sudo reboot`. Once rebooted, you should see the game systems appear on the main menu alongside the original "RetroPie" menu.
+
+## Additional Setup Options
+
+* [Configure Wifi](Wifi) to allow for easy [Updating](Updating-RetroPie), downloading themes, and SSH
+* [Enable SSH](SSH) to connect to your Raspberry Pi remotely and make it easy for [Transferring ROMs](Transferring-Roms)
+* Configure more controllers. This can be done after plugging in the new controller and pressing *Start* on your controller and selecting *Configure Input*
+* Install additional emulators/cores. From the RetroPie main menu, select *RetroPie Setup > Manage Packages > Manage Optional Packages*. Stable additional packages are available here with more unstable packages listed in the *Manage experimental packages* section. The packages are ordered first by type (emulators / libretro cores / ports), then alphabetically. By selecting a package you can choose to install it or remove it. Some packages also have additional configurations.
+  * It is recommend to install emulators in the *RetroPie - libretrocores* section if you wish to retain hotkey functionality (saving states, rewinding, etc) as hotkeys will usually only work with these types of emulators. Libretro emulators have names that begin with "lr-", such as "lr-flycast" or "lr-mame2000".
+* Expand the power of your hotkeys (libretro emulators only). Additional features are available but not enabled by default, such as rewinding and fast-forwarding. From the RetroPie main menu, select *Retroarch*. In the following menu, select *Settings > Input > Hotkey binds*. Press **Y (West)** to erase a bind. Press **A (East)** to define a new button.
```

Comment:
I don't agree with this kind of niche customisation on this page

Reply:
I used ZSNES all the time years ago, and my favorite feature was fast-forwarding through Final Fantasy 3/6 battles to reduce the grind. Not having it anywhere feels like a major hit to nostalgia when it's so useful for getting through long-winded intro cutscenes or traveling from A to B in RPG games. I'm totally cool with moving it to the [Hotkeys](https://retropie.org.uk/docs/RetroArch-Configuration/#hotkeys) page, though, and I can just link to it from here. How do you feel about that?

Reply:
Even in the hotkeys page, it would need caveats about the performance requirements, and some analysis about what hardware/systems can handle it or not, plus to turn it on i would recommend via https://retropie.org.uk/docs/Configuration-Editor/

so I think I am not convinced about adding it at the moment, in this form.